### PR TITLE
Described how workspace score is calculated

### PR DESCRIPTION
**Pull request by cwarnermm**
Added a section to the Workspace Optimization product documentation to describe how the overall workspace score is calculated.

### CODE REVIEW

File: source/configure/optimize-your-workspace.rst [hunk 1]
```diff
@@ -25,6 +25,15 @@ System Admins can access their workspace optimization page in the System Console
...
   :alt: Review your workspace health and growth scores, then take advantage of recommended optimizations.
 
+How is the overall score calculated?
+------------------------------------
+
+The highest score possible is 100. Your score is calculated based on the type of issue reported and the level of potential security risk introduced to your Mattermost deployment if ignored. 
+
+Each item on the dashboard is calculated based on its individual impact score which differ depending on whether they are suggestions, warnings, or problems. For example, if SSL encryption isn't configured in your workspace, Mattermost reports that as a problem, and that problem reduces your score until that problem is addressed. Suggestions impact your score less than warnings, and warnings impact less than problems.
+
+Want to improve your overall workspace optimization score? Take action towards the the suggestions, warnings, and problems reported on this dashboard. We recommend the following workspace optimizations.
```
Comment (michelengelen) [inline source/configure/optimize-your-workspace.rst:35]:
```suggestion
Want to improve your overall workspace optimization score? Take action towards the suggestions, warnings, and problems reported on this dashboard. We recommend the following workspace optimizations.
```

**Review by michelengelen — APPROVED** [review]
Just a minor thing!

**Review by laneycs — APPROVED** [review]
[👍🏽](https://emojipedia.org/thumbs-up-medium-skin-tone/) Looks good!

File: source/configure/optimize-your-workspace.rst [hunk 2]
```diff
@@ -25,6 +25,15 @@ System Admins can access their workspace optimization page in the System Console
 .. image:: ../images/workspace-optimization.png
   :alt: Review your workspace health and growth scores, then take advantage of recommended optimizations.
 
+How is the overall score calculated?
+------------------------------------
+
+The highest score possible is 100. Your score is calculated based on the type of issue reported and the level of potential security risk introduced to your Mattermost deployment if ignored. 
+
+Each item on the dashboard is calculated based on its individual impact score which differ depending on whether they are suggestions, warnings, or problems. For example, if SSL encryption isn't configured in your workspace, Mattermost reports that as a problem, and that problem reduces your score until that problem is addressed. Suggestions impact your score less than warnings, and warnings impact less than problems.
```
Comment (justinegeffen) [inline source/configure/optimize-your-workspace.rst:33]:
```suggestion
Each item on the dashboard is calculated based on its individual impact score. These differ depending on whether they're problems, warnings, or suggestions. For example, if SSL encryption isn't configured in your workspace, Mattermost reports that as a problem, which reduces your score until it's addressed. 

Warnings impact your score less than problems, and suggestions have the least impact on your score.
```

File: source/configure/optimize-your-workspace.rst [hunk 3]
```diff
@@ -25,6 +25,15 @@ System Admins can access their workspace optimization page in the System Console
...
+
+Want to improve your overall workspace optimization score? Take action towards the suggestions, warnings, and problems reported on this dashboard. We recommend the following workspace optimizations.
```
Comment (justinegeffen) [inline source/configure/optimize-your-workspace.rst:35]:
```suggestion
Want to improve your overall workspace optimization score? Take action towards the problems, warnings, and suggestions reported on this dashboard. We recommend the following workspace optimizations.
```

File: source/configure/optimize-your-workspace.rst [hunk 4]
```diff
@@ -55,4 +64,4 @@ The following optimization areas can alert you to workspace suggestions, warning
 |                       |   automated deactivations, and role assignments?                                                         |                                                                                                                                                                      |
 |                       |                                                                                                          | - `AD/LDAP <https://docs.mattermost.com/configure/configuration-settings.html#ad-ldap>`__                                                                            |
 |                       | - **Guest Accounts**: Do you want to control user access to channels and teams with guest accounts?      | - `Guest Accounts <https://docs.mattermost.com/onboard/guest-accounts.html>`__                                                                                       |
```
Comment (justinegeffen) [inline source/configure/optimize-your-workspace.rst:66]:
```suggestion
|                       | - **Guest accounts**: Do you want to control user access to channels and teams with guest accounts?      | - `Guest accounts <https://docs.mattermost.com/onboard/guest-accounts.html>`__                                                                                       |
```

**Review by justinegeffen — COMMENTED** [review]
Nice addition!

I reworded it and changed the order of score types for consistency.

**Comment by cwarnermm** [comment]
Really great editorial improvements, @justinegeffen! Thank you!!

**Pull request merged**